Mathlib/LinearAlgebra/Matrix/ProjectiveSpecialLinearGroup.lean

```diff
@@ -31,3 +31,55 @@ abbrev ProjectiveSpecialLinearGroup : Type _ :=
 
 /-- `PSL(n, R)` is the projective special linear group `SL(n, R)/Z(SL(n, R))`.-/
 scoped[MatrixGroups] notation "PSL(" n ", " R ")" => Matrix.ProjectiveSpecialLinearGroup (Fin n) R
+
+variable {n : Type u} [DecidableEq n] [Fintype n] {R : Type v} [CommRing R]
+
+namespace SpecialLinearGroup
+
+open SpecialLinearGroup
+
+/-- This is a converter to help you with `quotient.liftOn'` and `quotient.liftOn₂'` for `PSL(n, R)`.
+
+Example :
+```lean
+instance : SMul PSL(2, ℝ) ℍ where
```

Comment:
I think it would be good to develop more general machinery for defining these instances. Whenever you have a group `G` acting on `X`, and a normal subgroup `N` of `G` that acts trivially on `X` (via the induced action), then `G/N` inherits an action on `X`.

Would you mind writing that API instead?

Reply:
@jcommelin Thanks. I have opened the PR #11212 for this.